test/e2e/runner.go

```diff
@@ -215,8 +215,10 @@ func teardown() {
 			log.Printf("cannot fetch openshift metrics: %v", err)
 		}
 	}
-	if err := cliProvisioner.FetchActivityLog(acct, logsPath); err != nil {
-		log.Printf("cannot fetch the activity log: %v", err)
+	if !cfg.SkipLogsCollection {
```

Comment:
Somehow this got missed when the `SkipLogsCollection` config option was added

Reply:
Bonus bug fix.